__tests__/index.test.js

```diff
@@ -154,3 +201,40 @@ testIssueComment('/hi', async (context) => {
         {"body": "Hi @statler and waldorf!" }
     ])
 })
+
+let mockGetInstallationIDForRepo = jest.fn(() => 'installation-id')
+jest.mock('../GitForWindowsHelper/get-installation-id-for-repo', () => {
+    return mockGetInstallationIDForRepo
+})
+
+let mockSearchIssues = jest.fn(() => [])
+jest.mock('../GitForWindowsHelper/search', () => {
+    return {
+        searchIssues: mockSearchIssues
+    }
+})
+
+testIssueComment('/open pr', {
+    issue: {
+        number: 4281,
+        title: '[New gnutls version] GnuTLS 3.8.0',
+        body: `Released a bug-fix and enhancement release on the 3.8.x branch.[GnuTLS 3.8.0](https://lists.gnupg.org/pipermail/gnutls-help/2023-February/004816.html)
+
+Added the security advisory.[GNUTLS-SA-2020-07-14](security-new.html#GNUTLS-SA-2020-07-14)
+
+http://www.gnutls.org/news.html#2023-02-10`
+    }
+}, async (context) => {
+    expect(await index(context, context.req)).toBeUndefined()
+    expect(context.res).toEqual({
+        body: `I edited the comment: appended-comment-body-existing comment body
+
+The MINGW workflow run [was started](dispatched-workflow-open-pr.yml)`,
+        headers: undefined,
+        status: undefined
+    })
+    expect(mockGetInstallationAccessToken).toHaveBeenCalledTimes(1)
+    expect(mockGitHubApiRequestAsApp).not.toHaveBeenCalled()
+    expect(dispatchedWorkflows).toHaveLength(2)
+    expect(dispatchedWorkflows.map(e => e.payload.inputs.package)).toEqual(['mingw-w64-gnutls', 'gnutls'])
```

Comment:
Since we're expecting two dispatched workflows, shouldn't the comment body mention both?

Reply:
I've added 5635fce742dea24d45eb2b32eca0593ad36f7876 to get _some_ coverage going. What do you think @rimrul?

Reply:
Oh, right. That's two separate senzences in two different edits. I confused it with `/deploy` which has multiple workflows added to the comment at once.